src/swap/changelly.js

```diff
@@ -255,7 +272,9 @@ export function makeChangellyPlugin (
       const quoteInfo: FixedQuoteInfo = sendReply.result
       const spendInfoAmount = await request.fromWallet.denominationToNative(
         quoteInfo.amountExpectedFrom,
-        quoteInfo.currencyFrom.toUpperCase()
+        // need to verify that this is okay
+        // why use currencyCode from quoteInfo in the first place?
+        request.fromCurrencyCode.toUpperCase()
```

Comment:
Yes, this looks OK. I don't know why Allen did it this way, since the request version is the official one.